include/common/el3_common_macros.S

```diff
@@ -70,8 +70,15 @@
 	isb
 
 	/* ---------------------------------------------------------------------
-	 * Enable the SError interrupt now that the exception vectors have been
-	 * setup.
+	 * Early set RES1 bits in SCR_EL3. Set EA bit as well to catch both
+	 * synchronous and asynchronous exceptions in EL3.
```

Comment:
Sorry to be pedantic, but I think it's really important we get the language here correct. This affects aborts, not all exception types. Also, you use the term "SError" in some places, and "asynchronous external abort" in others without saying they're the same thing. Suggest saying here "Set EA bit as well to catch External Aborts and SError Interrupts in EL3". In the commit message you could say something like "SError Interrupts are also known as asynchronous external aborts".